forgot_password/handlers/util/email.py

```diff
@@ -25,14 +25,25 @@ def __init__(self, **smtp_params):
     def send_mail(self, sender, to, subject, text, html=None, reply_to=None):
         """
         Send email to user.
+
+        Arguments:
+        sender - (tuple) a tuple of the form ('Name', 'sender@example.com')
```

Comment:
While the underlying library expects the the sender to be a tuple (just the `sender` has to be a tuple, this doesn’t apply the same to `reply_to`), I think we should check the type of the parameters here, and convert a simple string to a tuple. This helps user of this util as giving a string param is much easier.